broker/l2tp_broker.cfg.example

```diff
@@ -12,6 +12,8 @@ tunnel_id_base=100
 ; Reject connections if there are less than N seconds since the last connection.
 ; Can be less than a second (e.g., 0.1).
 connection_rate_limit=10
+connection_rate_limit_per_ip=1
```

Comment:
First of all, each option here should come with a comment explaining it.
Secondly, this particular configuration makes little sense, does it? With a general rate limit of 10, an additional limit of 1 per client will never even kick in.

Also see https://github.com/wlanslovenija/tunneldigger/pull/137 where I propose to change the default settings.

Reply:
Modified in last commit.